tools/duckduckgo/ddg.go

```diff
@@ -14,10 +14,9 @@ type Tool struct {
 
 var _ tools.Tool = Tool{}
 
-func New() (*Tool, error) {
-	maxResults := 5
+func New(maxResults int, userAgent string) (*Tool, error) {
```

Comment:
should be documented.

it doesn't seem all that important to supply a user agent - can you export a default user agent so that folks can reach for it?